plugin/activity-log.vim

```diff
@@ -159,25 +174,25 @@ endfunction
 
 
 
-function TimeLogLine(action)
-	let l:file = expand("%:p")
-	let l:time = strftime('%Y-%m-%d %H:%M:%S')
-	let l:task = getline('.')
-	let l:task = substitute(l:task,"  ","","g") 
-	let l:branch = ''
-
-	if g:activity_log_append_git_branch
-		let l:branch = system('cd ' . expand("%:h") . "; git branch --no-color 2> /dev/null | sed -e '/^[^*]/d'")
-		if (l:branch =~ "^* ")
-			let l:branch = substitute(l:branch, '\* ', '', '')
-			let l:branch = substitute(l:branch, '\n', '', '')
-		endif
-	endif
+" function TimeLogLine(action)
```

Comment:
If you only want that one commit:

    git reset --hard 82c2fa4
    git push -f